src/components/FaucetButton.tsx

```diff
@@ -1,24 +1,27 @@
 import { ExternalLinkIcon } from '@heroicons/react/outline';
 import { Trans } from '@lingui/macro';
 import { Button, SvgIcon, Typography } from '@mui/material';
-import { useRouter } from 'next/router';
 import { useProtocolDataContext } from 'src/hooks/useProtocolDataContext';
 import { DarkTooltip } from './infoTooltips/DarkTooltip';
-import { ROUTES } from './primitives/Link';
+import { Link, ROUTES } from './primitives/Link';
 
 export const FaucetButton = () => {
   const { currentNetworkConfig } = useProtocolDataContext();
-  const router = useRouter();
 
   return (
     <DarkTooltip title="Get free assets to test the Aave Protocol">
-      <Button variant="outlined" size="small" onClick={() => router.push(ROUTES.faucet)}>
-        <Typography sx={{ display: 'inline-flex', alignItems: 'center' }} variant="buttonS">
+      <Button component={Link} href={ROUTES.faucet} variant="outlined" size="small">
+        <img
+          src={currentNetworkConfig.networkLogoPath}
+          alt={currentNetworkConfig.name}
+          style={{ width: 14, height: 14 }}
+        />
```

Comment:
Would something like the `startIcon` prop from MUI's Button work for the image?  Or implementing it similarly to how `TokenIcon` does it, which essentially wraps it in either an `Icon` component.

Simply `<Button startIcon={<img src="" />` maybe or wrapping it with an `Icon` component, or is this over-engineering something as simple as this direct img, which works as is?

Reply:
I tried to use the start and end icon props, but was having trouble sizing the icon correctly. They were too big. Let me take a look at `TokenIcon` and `GetABPToken`, see if there's a cleaner way.

Reply:
Updated with to use `startIcon` and `endIcon`. I had to explicitly set the height and width on the icon in order to size it properly.